eng/scripts/automation/Invoke-CadlDataPlaneGenerateSDKPackage.ps1

```diff
@@ -45,24 +39,57 @@ Invoke-CadlDataPlaneGenerateSDKPackage.ps1 -service anomalydetector -namespace A
 
 #>
 param (
-  [string]$service,
-  [string]$namespace,
-  [string]$sdkPath,
-  [string]$cadlRelativeFolder,
+  [string]$sdkFolder,
+  [string]$cadlSpecDirectory,
   [string]$specRoot = "",
-  [string]$repo="",
-  [string]$commit="",
-  [string]$additionalSubDirectories="" #additional directories needed, separated by semicolon if more than one
+  [string]$repo = "Azure/azure-rest-api-specs",
+  [string]$commit = "",
+  [string]$additionalSubDirectories="", #additional directories needed, separated by semicolon if more than one
+  [switch]$help
 )
+if ($help) {
+  Write-Host("Usage:
+Invoke-CadlDataPlaneGenerateSDKPackage.ps1 -sdkFolder <sdk-folder-path> -cadlSpecDirectory <relativeCadlProjectFolderPath> [-commit <commitId>] [-repo <specRepo>] [-specRoot <specRepoRootPath>] [-additionalSubDirectories <relativeFolders>]
+
+Options:
+-sdkFolder [Required] take the address of the sdk folder in azure-sdk-for-net repo. e.g. /home/azure-sdk-for-net/sdk/anomalyDetector/Azure.AI.AnomalyDetector
+-cadlSpecDirectory [Required] takes the relative path of the cadl project folder in spec repo. e.g. specification/cognitiveservices/AnomalyDetector
+-additionalSubDirectories [Optional] takes the relative paths of the additional directories needed by the cadl project
+-commit takes the git commit hash  (e.g. ac8e06a2ed0fc1c54663c98f12c8a073f8026b90)
+-repo [Optional] takes the `<owner>/<repo>` of the REST API specification repository. (e.g. Azure/azure-rest-api-specs). The default is Azure/azure-rest-api-specs
+-specRoot takes the file system path of the spec repo. e.g. /home/azure-rest-api-specs
+
+hint: You need to provide the cadl project path either (`-commit`, `-repo`) pair to refer to an URL path of the cadl project or `-specRoot` to refer to local file system path. If you provide both, `-specRoot` will be ignored.
```

Comment:
remove this, neither are needed we can create an empty cadl-location.yaml with values like <FILL IN> to indicate they need to fill the value in.  Or even put the documentation string like this

`commit: <takes the git commit hash  (e.g. ac8e06a2ed0fc1c54663c98f12c8a073f8026b90)>`

Reply:
removed